src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
It's recommended to use the name() method instead of toString() when you want to get the String representation of an enum constant. If the toString() method is not overridden in the Ball class, it will work similar to name(), but if it's overridden, it may return unexpected results. So, it's safer to use name() to avoid such issues.